Makefile

```diff
@@ -94,9 +94,9 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 .PHONY: test
-test: manifests generate fmt vet ## Run tests. Use WHAT=unit or WHAT=integration
+test: manifests generate fmt vet envtest ## Run tests. Use WHAT=unit or WHAT=integration
 ifeq ($(WHAT),integration)
-	go test -v ./test/integration/suites/... -coverprofile integration-cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_VERSION) --bin-dir $(LOCALBIN) -p path)" go test -v ./test/integration/suites/... -coverprofile integration-cover.out
```

Comment:
I just noticed this but aren't you basically changing the behavior of test to now implicitly always download envtest, and also run the integration tests always against envtest? 

I mean im not totally opposed to that but I was hoping that this would be a separate change as the specific CI step should be removed in that same notion: https://github.com/kro-run/kro/blob/main/.github/workflows/integration-tests.yaml#L31-L37

Reply:
https://github.com/kro-run/kro/pull/636